doc/api/http.md

```diff
@@ -917,6 +917,48 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaderNames()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing the names of the current outgoing headers. All
+header names are lowercase.
+
```

Comment:
I think it should be clearer what happens with duplicate header names, that they get collapsed after being lowercased.